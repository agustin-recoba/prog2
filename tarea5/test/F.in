insertarEnAvl 10
insertarEnAvl 8
insertarEnAvl 6
arregloAConjunto 4 2 3 4 6
unionDeConjuntos 3 3 6 9
estaLlenaPila
alturaDeAvl
avlMin 5
cardinalidad
perteneceAConjunto 2
perteneceAConjunto 9
iteradorDeConjunto
Fin
