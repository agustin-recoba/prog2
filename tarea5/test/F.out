1>Insertado.
2>Insertado.
3>Insertado.
4>Arreglo a conjunto con 4 elementos.
5>Union.
6>p NO estÃ¡ llena.
7>Altura de avl: 2.
8>
--(12,0.00)
-(11,0.00)
--(10,0.00)
---(9,0.00)
(8,0.00)
--(7,0.00)
---(6,0.00)
-(5,0.00)
---(4,0.00)
--(3,0.00)
---(2,0.00)
----(1,0.00)
9>5.
10>Pertenece.
11>Pertenece.
12>2 3 4 6 9 
13>Fin.
